### ISSUE TITLE
App navigation bar disappears and can't be restored

### ISSUE THREAD

**Issue by shawncrawley** [msouff/nwm-forecast-viewer#5]
So when I resize my window, the nav bar disappears and I can no longer do anything. The hamburger menu icon that (in other apps) triggers the hiding and showing of the navigation bar was removed. Tethys is programmed to automatically close the navigation bar if the window is too small, so it seems to me that the hamburger menu icon and its functionality should still be a part of the app as a safeguard, if nothing more.

**Comment by msouff** [comment]
Yes, I had it hidden by default. It needs to be changed to display when the screen size or zoom change.
